src/main/resources/edu/cornell/kfs/module/cg/businessobject/datadictionary/AwardStatus.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?><beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p" xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+<!--
+ Copyright 2005-2008 The Kuali Foundation
```

Comment:
Should this use the license header from the more recent KualiCo releases instead?